README.md

```diff
@@ -0,0 +1,34 @@
+# Comit Javascript SDK
+
+A software development kit for developing Javascript applications on top of the COMIT protocol reference implementation [comit-rs](https://github.com/comit-network/comit-rs).
+
+Currently [comit-rs](https://github.com/comit-network/comit-rs) is shipped as a binary called [cnd](https://github.com/comit-network/comit-rs/releases) (Comit Node Daemon).
```

Comment:
Initially, we said it is the `comit network daemon` because `node` and `daemon` are kind of synonyms.

```suggestion
Currently [comit-rs](https://github.com/comit-network/comit-rs) is shipped as a binary called [cnd](https://github.com/comit-network/comit-rs/releases) (Comit Network Daemon).
```

Reply:
> Initially, we said it is the `comit network daemon` because `node` and `daemon` are kind of synonyms.

I want written proof :)